include/seqan3/range/views/minimiser.hpp

```diff
@@ -353,10 +352,13 @@ class minimiser_view<urng_t>::window_iterator
     //!\brief Calculates minimisers for the first window.
     void window_first(uint32_t window_values_size)
     {
-        for (uint32_t i = 0; (i < window_values_size - 1) ; i++)
+        if (window_values_size == 0u)
```

Comment:
This looks more like a semantic fix rather than just code-style. Maybe you want to rename the commit message?

Reply:
I'll fix the commit message at the end, okay?